### PR TITLE
Add cyberdyne takeover and engage

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added a takeover and engage page for cyberdyne

## QA
- ./run
- http://0.0.0.0:8012/
- Refresh until you get the cyberdyne takeover
- Follow CTA to engage page
- Test form

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/7337

### CODE REVIEW

File: templates/engage/cyberdyne.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <p>ケーススタディでは、以下を含む詳細をご紹介します。</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Snapとブランドストアによる運用コスト（OPEX）とエンジニアリング経費の削減</li>
+        <li class="p-list__item is-ticked">Snapの差分更新による必要帯域幅の95%削減</li>
+        <li class="p-list__item is-ticked">市場投入までの時間の短縮と新機能リリース</li>
+      </ul>
+    </div>
+    <div class="col-5" id="register-section">
+      <h3>ダウンロード</h3>
+      {% with  id="3241", returnURL="https://pages.ubuntu.com/rs/066-EOV-335/images/OpenStack-made-easy_eBook_jp.pdf" %}
```
Comment (pmahnke) [inline templates/engage/cyberdyne.html:33]: I don't think this is the correct ebook... not 100% sure.  Otherwise it is fine.

Comment (deadlight) [reply templates/engage/cyberdyne.html:33]: @pmahnke whoops... yeah, that needed updating - I submitted this too soon. I'm just waiting for the thank-you page.

**Comment by deadlight** [comment]
@pmahnke ok, I've added the form and thank you page now

File: templates/engage/cyberdyne.html (hunk 1 above)
Comment (pmahnke) [reply templates/engage/cyberdyne.html]: so I think that's the right landing page, but that page (Marketo) doesn't have the pdf and sends you to ubuntu.com, if you want to pass that along.

**Comment by SirSamTumless** [comment]
@deadlight PDF can be found here; https://pages.ubuntu.com/rs/066-EOV-335/images/Cyberdine_JP_15.05.20%202.pdf

**Pull request merged**